zenodo/modules/deposit/templates/zenodo_deposit/delete.html

```diff
@@ -44,41 +48,33 @@ <h1>Delete record</h1>
         <p><h4>Are you ABSOLUTELY sure?</h4> This action <strong>CANNOT</strong> be undone! Deleting a record will
         permanently delete the associated files from disk.</p>
       </div>
+      {% if record_past_deletion_time %}
+        <div class="alert alert-warning" style="background-color:#df6d69" align="center"
+          <p> <h3 style="color:white"> ⚠️ This record is 1 month old! ⚠️ </h3> </p>
+        </div>
+      {% endif %}
     </div>
   </div>
   <div class="row">
     <div class="col-md-12">
       <div class="panel panel-default">
         <div class="panel-heading">Record and associated objects</div>
-        <ul class="list-group">
+          <div class="panel-body">
+          <h5>Owners</h5>
+          <ul class="list-group">
           {% if owners %}
-            <li class="list-group-item">
-              <h5>Owners</h5>
-              <table class="table table-bordered">
-                <tr>
-                  <th>User ID</th>
-                  <th>Email</th>
-                  <th>Username</th>
-                  <th>Full Name</th>
-                </tr>
-                {%- for owner in owners %}
-                <tr>
-                  <td>{{owner.id}}</td>
-                  <td>{{owner.email}}</td>
-                  <td>{{owner.profile and owner.profile.username}}</td>
-                  <td>{{owner.profile and owner.profile.full_name}}</td>
-                </tr>
-                {%- endfor %}
-              </table>
-            </li>
+            {%- for owner in owners %}
+              {{ user_info_table(owner) }}
+            {%- endfor %}
           {% endif %}
+          </div>
           <li class="list-group-item">
-            <h5>UUID</h5>
-            {{record.id}}, revision {{record.revision_id}} (created: {{record.created}}, updated: {{record.updated}})
+            <h5>Record</h5>
+              {{ record_info_table(record, owners) }}
```

Comment:
```suggestion
              {{ record_info_table(record) }}
```
I'm not sure if `owners` is needed actually, so I would remove the parameter.